src/06-errors_and_files/03-exceptions.md

```diff
@@ -78,7 +78,7 @@ main =
 >    Features such as syntactic extensions (like LambdaCase above), extensions to the type checker,
 >    and more.
 >
->    These extensions can be added by adding `{-# language <extension-name> #-}`
+>    These extensions can be added by adding `{-# LANGUAGE <extension-name> #-}`
```

Comment:
```suggestion
>    These extensions can be added by adding `{-# language <extension-name> #-}`
```

Reply:
Ditto

Reply:
I've updated the suggestion to include a note about `language`.